app/src/flavor-adblocker/kotlin/adblocker/TunnelDashes.kt

```diff
@@ -15,7 +16,7 @@ class TunnelDashCountDropped(
 ) : Dash("tunnel_drop",
         R.drawable.ic_block,
         ctx.getString(R.string.tunnel_dropped_count_desc),
-        onLongClick = { t.tunnelDropCount %= 0; t.tunnelDropStart %= System.currentTimeMillis(); true }
```

Comment:
Honestly, I just didn't like the share feature. I probably would have left it alone had it been in the advance options list instead of homepage. But it takes up space on the homepage, and honestly, what good do you think it will do to post on social media sites that the advertisements were blocked on those sites? If anything that action is just begging developers to find a way to prevent blokada from working.

Reply:
First of all you are ignoring the most important part which is that your branch is behind master.
Second you are removing features only on the basis of your own opinion. You didn't start a discussion or asked anyone what they think about it.
To make things worse your commit dosn't completely reverse the features that where added but just break them.
Generally speaking you should only do a PR for one group of thematically related commits.
For example this PR could be split in "Share button removed", "cleanup of depricated functions and renaming" and "wildcards added".  But the most minimal thing to expect would be that **ALL** changes are listed in the PR description.
If you would have done it like this it would be much faster to make code reviews and it would be simpler to discuss changes.

Reply:
> First of all you are ignoring the most important part which is that your branch is behind master.

Is there a way to have the updates to master automatically apply to the branch without undoing the code changes to the branch?

> Second you are removing features only on the basis of your own opinion. You didn't start a discussion or asked anyone what they think about it.

I didn't start a conversation because it is fine to leave it in. I wasn't trying to remove them from the master. 

> Generally speaking you should only do a PR for one group of thematically related commits.
For example this PR could be split in "Share button removed", "cleanup of depricated

Yes. I felt the same. It just took so long to find the files for the specific change, then take out the other code that wasn't relevant to the specific change. For example, in this line of code

`if (blockade.inwhitelist(host) || (!blockade.inblacklist(host) && !blockade.inwildcardlist(host))) {`

I changed "allow" to "inwhitelist" and "deny" to "inblacklist" which made the code alot clearer. Those changes required multiple files changed. I also added wildcard functionality. Had I known to do comments as you changed code, it would have been easier to document these changes correctly instead of going back when everything is working and try to undo changes for specific comments. 

 Now that I know how the pull request comments works it will be easier to make changes as they come in the future rather than building up a lot of changes, and trying to do one pull request instead of multiple. 

I apologize for not knowing the best practice of documentation when I was working on these changes.

Reply:
> Is there a way to have the updates to master automatically apply to the branch without undoing the code changes to the branch?

git rebase